hazelcast/src/test/java/com/hazelcast/quorum/cache/CacheQuorumListenerTest.java

```diff
@@ -166,14 +160,15 @@ public void onChange(QuorumEvent quorumEvent) {
             threeNode.put(generateKeyOwnedBy(h1), "bar");
             fail();
         } catch (Exception e) {
+            EmptyStatement.ignore(e);
         }
         try {
             fourNode.put(generateKeyOwnedBy(h1), "bar");
             fail();
         } catch (Exception e) {
+            EmptyStatement.ignore(e);
```

Comment:
rename the exception to expected and pleace remove the EmptyStatement.ignore. We do this for production code; but we should not do this for test code.